integrations/microstream/core/src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -0,0 +1,516 @@
+[
+{
+  "name":"one.microstream.afs.nio.types.NioReadableFile$Default[]"
```

Comment:
There is a lot of reflection configuration here, yet the feature is marked as not supporting native image.

Reply:
I set the native-image support to false because not all Microstream features, especially the cache, will work with native images. There are some Microstream modification required to get the cache working. Until those are release the provided configuration will allow to use the storage within native images.
If you prefer, I can remove all those configurations.